lib/convert-twig-includes-paths.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var utils = require('./utils');
+
+
+
+/*
+ * Convert file paths in twig include statements
+ *
+ * @param {Object} options  pattern importer options
+ * @param {String} twigContent  content from a twig file
+ *
+ * @returns {String} twigContent twig file content with converted paths
+*/
+function convertTwigIncludes (options,twigContent) {
+
+  if(!options.convertCategoryTitles){
+    return twigContent;
+  }
+
+  // regex expression to find twig include code blocks
+  var twigIncludeRegex = /{%.*include.*%}/g;
```

Comment:
you need to add greediness to this expression: `/{%.*?include.*?%}/g`
Example:
```twig
{% include 'index.html' %}
There is the line that conains '%}' and some text after that.
```
this what will be matched:
```twig
{% include 'index.html' %}
There is the line that conains '%}
```
instead of:
```twig
{% include 'index.html' %}
```